install_config/cluster_metrics.adoc

```diff
@@ -331,217 +268,153 @@ service.
 In order for the browser running the web console to trust the connection through this route,
 it must trust the route's certificate. This can be accomplished by
 xref:metrics-using-secrets-byo-certs[providing your own certificates] signed by a trusted
-Certificate Authority. The Metric Deployer's secret allows you to pass your own certificates
+Certificate Authority. The `*openshift_metrics*` role allows you to specify your own certificates
 which it will then use when creating the route.
 
-If you do not wish to provide your own certificates, the router's default certificate will
-be used instead.
+The router's default certificate are used if you do not provide your own.
 
 [[metrics-using-secrets-byo-certs]]
 ==== Providing Your Own Certificates
 
 To provide your own certificate which will be used by the
 xref:../architecture/core_concepts/routes.adoc#secured-routes[re-encrypting route],
-you can pass these values as
-xref:../dev_guide/secrets.adoc#dev-guide-secrets[secrets] to the Metrics Deployer.
+you can set the 'cert, key, and ca' xref:../install_config/cluster_metrics.adoc#metrics-ansible-variables[variables]
+in your inventory file.
 
 The `hawkular-metrics.pem` value needs to contain the certificate in its *_.pem_*
 format. You may also need to provide the certificate for the Certificate Authority
 which signed this *_pem_* file via the `hawkular-metrics-ca.cert` secret.
```

Comment:
These are the files that would go into the secret for the deployer, no longer relevant.